docs/content/latest/reference/configuration/yugabyted/errors-glossary.md

```diff
@@ -0,0 +1,65 @@
+---
+title: yugabyted Errors and Warnings
+linkTitle: Errors and Warnings
+description: yugabyted CLI errors and warnings
+section: REFERENCE
+menu:
+  latest:
+    identifier: yugabyted-errors-warnings
+    parent: configuration
+    weight: 2455
+isTocNested: true
+showAsideToc: true
+---
+
+This page explains errors and warnings that you may experience while running a YugabyteDB cluster using yugabyted CLI and ways to resolve them.
+
+| CLI Message | CLI Command (Task) | Error/Warning |
+| --- | --- | --- |
+| [open files ulimits value set low](#open-files-ulimits-value-set-low) | installation | Warning |
+| [max user processes value set low](#max-user-processes-value-set-low) | installation | Warning |
+| [Transparent hugepages disabled](#transparent-hugepages-disabled) | installation | Warning |
+| [Six loopback IPs are required](#six-loopback-ips-are-required) | installation | Warning |
+| [ntp package missing](#ntp-package-missing) | installation | Warning |
+
+## open files ulimits value set low
+
+YugabyteDB needs to create a large number of files which may not be enabled/allowed by the OS by default.
```

Comment:
Let's add a little bit more description - "YugabyteDB needs to create a large number of files for writing data, logs, and a variety of other information onto the disk. Operating Systems (OS) may not be configured to allow a user process to open a large number of files. "